spec/requests/api/v2/providers/accredited_body_filter_providers_spec.rb

```diff
@@ -51,6 +60,21 @@ def perform_request
       get request_path, headers: { "HTTP_AUTHORIZATION" => credentials }
     end
 
+    context "when accredited_provider has a course" do
+      before do
+        accredited_provider_course
+      end
+
+      let(:filters) { "" }
+
+      it "is returned" do
```

Comment:
I know this is nit-picky, but I feel like you and I understand this, but it won't be obvious at all to others. Also, `it` in this context is the response from `GET /providers`. Could we make this a little more explicit:

```suggestion
      it "returns the accredited provider in the results" do
```